centreon/packages/ui/src/components/Modal/Modal.styles.ts

```diff
@@ -35,8 +35,8 @@ const useStyles = makeStyles<{
       transition: theme.transitions.create('left')
     },
     '&[data-size="large"] .MuiDialog-paper': {
-      maxWidth: '604px',
-      width: '604px'
+      maxWidth: '704px',
+      width: '704px'
```

Comment:
Be careful, this will have impact on others modal. Have you tried other large modal before?